mlrun/model_monitoring/applications/_application_steps.py

```diff
@@ -164,3 +165,39 @@ def _create_mlrun_context(app_name: str):
         )
         context.__class__ = MonitoringApplicationContext
         return context
+
+
+class _ApplicationErrorHandler(StepToDict):
+    def __init__(self, project: str, name: Optional[str] = None):
+        self.project = project
+        self.name = name or "ApplicationErrorHandler"
+
+    def do(self, event):
+        """
+        Handle model monitoring application error. This step will generate an event, describing the error.
+
+        :param event: Application event.
+        """
+
+        logger.error(f"Error in application step: {event}")
```

Comment:
I think this log should be debug/info, an error log has already been raised in the previous MM application step.
Here it's in the normal flow of the step.

Reply:
amm you right but the user has no idea that he is in the "error step" so I think `logger.error` provides a good indication for post analysis when searching for the relevant logs.